release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -858,6 +858,16 @@ clouds:
 
 To use application credentials with your cluster as a {rh-openstack} administrator, create the credentials. Then, use them in a `clouds.yaml` file when you install a cluster. Alternatively, you can create the `clouds.yaml` file and rotate it into an existing cluster.
 
+[id="ocp-4-12-hcp"]
+=== Hosted control planes (Technology Preview)
+
+[id="ocp-4-12-hcp-versioning"]
+==== Versioning for hosted control planes
+
+With each major, minor, or patch version release of {product-title}, 2 components of hosted control planes are released: the HyperShift Operator and the command-line interface (CLI). Any HyperShift Operator that is released for a minor _n_ version of {product-title} must work with _n_, _n-1_, and _n-2_ versions of {product-title}. However, because the hosted control planes feature was introduced as a Technology Preview in {product-title} version 4.11, it is currently available with versions only as far back as 4.11.
```

Comment:
```suggestion
With each major, minor, or patch version release of {product-title}, 2 components of hosted control planes are released: the HyperShift Operator and the command-line interface (CLI). Any HyperShift Operator that is released for a minor _n_ version of {product-title} must work with _n_, _n-1_, and _n-2_ versions of {product-title}. However, because the hosted control planes feature was introduced as a Technology Preview in {product-title} version 4.11, it is currently only available in versions 4.11 and above.
```
Contemplating removing the `currently` as well becuase I think the way it reads implies that there might be a back port in the future. Wdyt, also do you know if there might be a back port?

Reply:
Agree! I'll remove "currently". Great catch.